common/src/main/java/com/wynntils/screens/OverlaySelectionScreen.java

```diff
@@ -32,7 +32,8 @@ protected void init() {
     public void render(PoseStack poseStack, int mouseX, int mouseY, float partialTick) {
         poseStack.pushPose();
 
-        this.fillGradient(poseStack, 0, 0, this.width, this.height, -1072689136, -804253680);
+        int backgroundColor = CommonColors.DARK_GRAY.withAlpha(200).asInt();
+        this.fillGradient(poseStack, 0, 0, this.width, this.height, backgroundColor, backgroundColor);
```

Comment:
Is this correct? Now the gradient goes from backgroundColor to backgroundColor? Previously it was two different colors...

Reply:
Those two colors were basically the same, just some encoding stuff (probably happened while obfuscation). This does the same thing.